src/assets/lang/fr.json

```diff
@@ -22,5 +22,11 @@
   "formulaNotComputableText":"pas calculable",
   "editThisVariable":"Modifier ces informations",
   "editThisFormula":"Modifier cette formule",
-  "thisVariableJson":"Donnée brute au format JSON"
+  "thisVariableJson":"Donnée brute au format JSON",
+  "checkChangelog":"Vérifiez l'orthographe de l'URL. Si ce lien fonctionnait et ne fonctionne plus, vérifiez le {changelogURLLink}.",
+  "notParamNotVariable":"' {inputValueRef} ' n'est ni un paramètre ni une variable d'OpenFisca.",
+  "pageDoesNotExist":"La page « /{inputValueRef} » n'existe pas.",
```

Comment:
This makes a double `/` on the not found pages
![image](https://user-images.githubusercontent.com/11834997/28933673-45b43b2c-787e-11e7-8b37-6483054e8a48.png)